### PR TITLE
SemanticUI for AdminArea

### LINKED ISSUE #146: Rewrite admin area to SemanticUI
Bootsrap4 sucks... looks ugly and poor design of tools force me rewrite UI to  http://semantic-ui.com/ with https://github.com/doabit/semantic-ui-sass

### PR DESCRIPTION
**Pull request by sveredyuk**
Fixed #146 

Just start of rewriting whole admin to SemanticUI.
Please, check some screenshots below:

### CODE REVIEW

File: app/controllers/admin/base_controller.rb [hunk 1]
```diff
@@ -3,14 +3,53 @@ class BaseController < ApplicationController
...
+
+    def flash_error
+      add_flash(:error)
     end
 
-    def flash_error(resource_name)
-      flash[:error] = t("#{resource_name.to_s}.notifications.#{params[:action]}.error")
+    def add_flash(type)
+      resource = params[:controller].split('/').last.singularize.capitalize
+      action   = "#{params[:action]}d"
```
Comment (denys-medynskyi) [inline app/controllers/admin/base_controller.rb]: will it give `destroyd` message ?

File: app/helpers/admin/dry_helper.rb [hunk 2]
```diff
@@ -1,19 +1,26 @@
 module Admin
   module DryHelper
     BOOL_BG_COLOR_CLASS = {
-      true  => 'bg-success',
-      false => 'bg-danger'
+      true  => 'green',
```
Comment (denys-medynskyi) [inline app/helpers/admin/dry_helper.rb:4]: do you like [green gray band](http://modna.com.ua/images/fact/green-grey-poedut-na-mtv-ema.jpg)  ?

**Review by denys-medynskyi — APPROVED** [review]
cool shit

File: app/controllers/admin/base_controller.rb (hunk 1 above)
Comment (sveredyuk) [reply app/controllers/admin/base_controller.rb]: I will add destroy message during rewriting whole admin area. more commits coming soon

**Pull request merged**